CMakeLists.txt

```diff
@@ -7,7 +7,7 @@ if(NOT Slicer_SOURCE_DIR)
   set(EXTENSION_CATEGORY "Surface Models")
   set(EXTENSION_CONTRIBUTORS "Jun LIN, Xiaojun CHEN, Ph.D (SJTU)")
   set(EXTENSION_DESCRIPTION "This is an extension module to set the osteotomy trajectory with multiple planes and clip with just one click.")
-  set(EXTENSION_ICONURL "https://raw.githubusercontent.com/Ting-Jia/ModelClip/master/Resources/Icons/ModelClip.png")
+  set(EXTENSION_ICONURL "https://raw.githubusercontent.com/jamesobutler/ModelClip/main/Resources/Icons/ModelClip.png")
   set(EXTENSION_SCREENSHOTURLS "https://raw.githubusercontent.com/Ting-Jia/ModelClip/master/ModelClipScreenShot.png")
```

Comment:
Can you also update the screenshot URL as I was doing in https://github.com/jamesobutler/ModelClip/commit/943f228272ee89fb50c5455849f2cdf4229a703f?

Reply:
I went ahead and committed the change in https://github.com/jamesobutler/ModelClip/commit/804eccc17a97e4ded3692c4cce2ebae3d35036e5.